src/pages/Encounters/EncounterHistorySelector.tsx

```diff
@@ -395,20 +396,21 @@ export default function EncounterHistorySelector() {
         <h2 className="px-2 mb-2 text-xs font-medium text-gray-600 uppercase">
           {t("chosen_encounter")}
         </h2>
-        <Sheet open={isOpen} onOpenChange={setIsOpen}>
-          <SheetTrigger className="w-full">
+        <Drawer open={isOpen} onOpenChange={setIsOpen}>
+          <DrawerTrigger className="w-full">
             <EncounterSheetTrigger />
-          </SheetTrigger>
-          <SheetContent
-            side="bottom"
-            className="max-h-[85vh] rounded-t-3xl overflow-y-auto mb-2"
-          >
-            <SheetHeader className="px-4 pb-2">
-              <SheetTitle>{t("past_encounters")}</SheetTitle>
-            </SheetHeader>
-            <EncounterHistoryList onSelect={() => setIsOpen(false)} />
-          </SheetContent>
-        </Sheet>
+          </DrawerTrigger>
+          <DrawerContent className="px-4">
+            <DrawerHeader className="py-1.5">
+              <DrawerTitle className="text-lg font-semibold">
+                {t("past_encounters")}
+              </DrawerTitle>
+            </DrawerHeader>
+            <div className="overflow-y-auto pb-4">
```

Comment:
Wouldn't be nice to have some padding between the div and scroll bar ?, currently it is overlapped on android
```suggestion
            <div className="overflow-y-auto pb-4 pr-1.5">
```